Sources/PostgresNIO/New/Messages/SASLInitialResponse.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 24.01.21.
+//
+
+extension PSQLFrontendMessage {
```

Comment:
Ditch Xcode template headers.
```suggestion
extension PSQLFrontendMessage {
```